src/SearchParameters/RefinementList.js

```diff
@@ -52,7 +52,9 @@ var lib = {
    */
   removeRefinement: function removeRefinement(refinementList, attribute, value) {
     if (value === undefined) {
-      return lib.clearRefinement(refinementList, attribute);
+      return lib.clearRefinement(refinementList, function(v, f) {
```

Comment:
We can add a comment that explain why we force the usage of the function.

Reply:
```suggestion
      // we use the "filter" form of clearRefinement, since it leaves empty values as-is
      // the form with a string will remove the attribute completely
      return lib.clearRefinement(refinementList, function(v, f) {
```